src/utils/events/event-manager.js

```diff
@@ -191,7 +213,7 @@ export default class EventManager {
     const alias = BASIC_EVENT_ALIASES[srcEvent.type];
     if (alias) {
       // fire all events aliased to srcEvent.type
-      const emitEvent = Object.assign({}, event, {type: alias});
+      const emitEvent = Object.assign({}, event, {isDown: true, type: alias});
```

Comment:
Looks like you're always setting `isDown: true` here, should the third `Object.assign()` param be the second?